script/DeployFactory.s.sol

```diff
@@ -7,10 +7,24 @@ import {SafeSingletonDeployer} from "safe-singleton-deployer-sol/src/SafeSinglet
 import {CoinbaseSmartWallet, CoinbaseSmartWalletFactory} from "../src/CoinbaseSmartWalletFactory.sol";
 
 contract DeployFactoryScript is Script {
-    address constant EXPECTED_IMPLEMENTATION = 0x000100abaad02f1cfC8Bbe32bD5a564817339E72;
-    address constant EXPECTED_FACTORY = 0x0BA5ED0c6AA8c49038F819E587E2633c4A9F428a;
+    // cast code 0x914d7Fec6aaC8cd542e72Bca78B30650d45643d7 --rpc-url https://mainnet.base.org
+    bytes constant SAFE_SINGLETON_FACTORY_CODE =
+        hex"7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3";
```

Comment:
ideally `SafeSingletonDeployer` would export this, but hardcoding it here for now

Reply:
yeah good idea to export

Reply:
or alternatively, maybe we should just have a new function like, "deployAnvil"? which uses etch and doesn't have to expose